test/rules/prefer-while/test.ts.fix

```diff
@@ -0,0 +1,14 @@
+// for loops without an initializer, termination condition, and incrementor should be updated
+while (true) {
+    console.log(x);
+}
+
+// for loops without an initializer and incrementor should be updated
+while (true===true) {
+    console.log(x);
+}
+
+// for loops with an initializer, termination condition, and incrementor should remain untouched
+for(let x = 1; x < 10; x++) {
+    console.log(x);
+}
```

Comment:
can we add a test case for when it's incremented with `x+=1`?
does the rule handle the case when the increment happens within the block? Example:
```
for (let i = 0; i < 10;) {
    i += 1;
}
```

Reply:
Yes and yes! Added test cases in 187bc3d